File: Div2A/CF709-D2-A.cpp
#include <bits/stdc++.h>

using namespace std;
int main() {
    long long n,b,d,x,sum(0), count(0);
    cin>>n>>b>>d;

    while(n>0) {
        cin>>x;
        if (x <= b) {
            sum += x;
        }
        if(sum>d){
            sum = 0;
            count++;
        }
        n--;
    }

    cout<<count;
}

File: Div2A/CF709-D2-A.py
n,b,d = [int(x) for x in input().split(" ")]
sum = 0
count = 0
x = [int(x) for x in input().split(" ")]
for i in range(0, len(x)):
    if x[i]<= b:
        sum += x[i]
    if sum>d:
        sum=0
        count +=1
print(count)